modules/olmv1-installing-an-operator.adoc

```diff
@@ -142,48 +160,95 @@ clusterextension.olm.operatorframework.io/pipelines-operator created
 $ oc get clusterextension pipelines-operator -o yaml
 ----
 +
---
-include::snippets/olmv1-version-range-cr-adomn.adoc[]
---
-+
 .Example output
 [%collapsible]
 ====
 [source,text]
 ----
-apiVersion: olm.operatorframework.io/v1alpha1
-kind: ClusterExtension
+apiVersion: v1
+items:
+- apiVersion: olm.operatorframework.io/v1alpha1
+  kind: ClusterExtension
+  metadata:
+    annotations:
+      kubectl.kubernetes.io/last-applied-configuration: |
+        {"apiVersion":"olm.operatorframework.io/v1alpha1","kind":"ClusterExtension","metadata":{"annotations":{},"name":"pipelines-operator"},"spec":{"channel":"latest","installNamespace":"openshift-marketplace","packageName":"openshift-pipelines-operator-rh","pollInterval":"30m"}}
+    creationTimestamp: "2024-06-10T17:50:51Z"
+    generation: 1
+    name: pipelines-operator
+    resourceVersion: "53324"
+    uid: c54237be-cde4-46d4-9b31-d0ec6acc19bf
+  spec:
+    channel: latest
+    installNamespace: openshift-marketplace
```

Comment:
@michaelryanpeter suggest openshift-operators, not openshift-marketplace.
per my understanding, openshift-marketplace is used for catsrc, while openshift-operators is used for operator installation.

should check it in the PR.

Reply:
@kuiwang02 Thank you for the feedback. Addressed in https://github.com/openshift/openshift-docs/commit/587f6bd25e3cad42aefa47f211480b2064cb1fb4.